modules/ossm-jaeger-service-mesh.adoc

```diff
@@ -0,0 +1,16 @@
+////
+This CONCEPT module included in the following assemblies:
+-assembly.adoc
+////
+
+[id="ossm-jaeger-service-mesh_{context}"]
+= Jaeger and service mesh
+
+Installing Jaeger via the Service Mesh on OpenShift differs from community Jaeger installations in multiple ways.  These modifications are sometimes necessary to resolve issues, provide additional features, or to handle differences when deploying on OpenShift or OKD.
```

Comment:
Jaeger is not installed via the service mesh - it needs the jaeger-operator to be pre-installed (and also the elasticsearch operator) - then the Service Mesh will create the CR for Jaeger.